extension/src/test/suite/experiments/index.test.ts

```diff
@@ -173,11 +173,11 @@ suite('Experiments Test Suite', () => {
   })
 
   describe('handleMessageFromWebview', () => {
-    after(() => {
+    after(() =>
       workspace
-        .getConfiguration(ConfigKey.EXP_TABLE_HEAD_MAX_DEPTH)
```

Comment:
[F] This was not behaving as expected. It kept leaving behind the config option after the suite had run. If you take a look at the other usage that we have of `workspace.getConfiguration()` it always looks like the new usage.